soccer/gameplay/plays/Attack_Goal/basic_122_attack.py

```diff
@@ -0,0 +1,198 @@
+import standard_play
+import behavior
+import skills
+import tactics
+import main
+import robocup
+import evaluation
+import constants
+import math
+import situational_play_selection
+
+
+class Basic122(standard_play.StandardPlay):
```

Comment:
I was thinking about changing the name of the ATTACK_GOAL basic_122 to be something like basic_122 attack, but didn't go through with it. I think it should be noted that this version has a narrower scope than the original but it seemed dumb to add "attack" to everything.

Reply:
I think if it's in the folder it should be sufficient to tell what its purpose is.